### PR TITLE
Read SCALECRS and ENDSCALE from EclipseState

### PR DESCRIPTION
**Pull request by jokva**
Requires https://github.com/OPM/opm-parser/pull/1020

### CODE REVIEW

File: opm/material/fluidmatrixinteractions/EclEpsConfig.hpp [hunk 1]
```diff
@@ -196,8 +196,9 @@ class EclEpsConfig
                       const Opm::EclipseState& eclState,
                       Opm::EclTwoPhaseSystemType twoPhaseSystemType)
     {
+        const auto& endscale = eclState.runspec().endpoint_scaling();
```
Comment (andlaus) [inline opm/material/fluidmatrixinteractions/EclEpsConfig.hpp]: isn't the `endpoint_scaling()` method name *quite* inconsistent with the rest of the parser?

Comment (jokva) [inline opm/material/fluidmatrixinteractions/EclEpsConfig.hpp]: Nothing is consistent in parser, but that'll hopefully be addressed soon.

Comment (andlaus) [inline opm/material/fluidmatrixinteractions/EclEpsConfig.hpp]: > Nothing is consistent in parse

that's a new function isn't it?

> but that'll hopefully be addressed soon.

I hope this is done in a way which makes the conventions used by opm-parser *more* similar to the other modules, not *less*!?

Comment (jokva) [inline opm/material/fluidmatrixinteractions/EclEpsConfig.hpp]: Maybe - the goal is a better interface.

It'll surely not happen before Deck is private.

**Comment by andlaus** [comment]
this probably needs a little fixup after OPM/opm-parser#1021? @jokva?

**Comment by andlaus** [comment]
this is superseded by #207...

**Pull request closed without merging**